### PR TITLE
Aggregate SirCAL messages by source and error

### LINKED ISSUE #464: Aggregate sir-complains-a-lot complaints by source
Currently we get too many individual complaints:

usa-facts confirmed_7dav_cumulative_prop (county, hrr, msa, state) has a 2-day gap of missing data in its most recent 7 days of data; last updated 2020-11-05.
usa-facts confirmed_7dav_incidence_num (county, hrr, msa, state) has a 2-day gap of missing data in its most recent 7 days of data; last updated 2020-11-05.
...

We can pull these together into a single line for the source:

usa-facts has a 2-day gap of missing data for N signals (confirmed_7dav_cumulative_prop, confirmed_7dav_incidence_num, etc.)

### PR DESCRIPTION
**Pull request by benjaminysmith**
### Description
Aggregate the complaints in sir-complains-a-lot by source and message

### Fixes 
- Fixes #464 

Example:
Screenshot

### CODE REVIEW

File: sir_complainsalot/delphi_sir_complainsalot/run.py [hunk 1]
```diff
@@ -57,28 +59,64 @@ def report_complaints(all_complaints, params):
...
+
+
+def format_complaints_aggregated_by_source(complaints):
+    """Build formatted Slack message for posting to the API, aggregating the 
+    complaints by source to reduce the number of blocks."""
+
+    blocks = [get_maintainers_block(complaints)]
+
+    def aggregated_message_for_source(x): return "{complaint} - (last update: {last_updated})".format(
+        complaint=x.message, last_updated=x.last_updated.strftime("%Y-%m-%d"))
```
Comment (krivard) [inline sir_complainsalot/delphi_sir_complainsalot/run.py:87]: While we're not yet including linting of this package in the repo CI, probably best not to introduce *new* linting errors

File: sir_complainsalot/delphi_sir_complainsalot/run.py [hunk 2]
```diff
@@ -57,28 +59,64 @@ def report_complaints(all_complaints, params):
...
+                    signal=complaint.signal, geo_types=", ".join(complaint.geo_types))
+            blocks.extend([
+                {
+                    "type": "divider"
+                },
+                {
+                    "type": "section",
+                    "text": {
+                        "type": "mrkdwn",
+                        "text": "*{source_name}* {message_for_group}:\n{signals}".format(source_name=source.upper(), message_for_group=message, signals=signal_and_geo_types)
```
Comment (krivard) [inline sir_complainsalot/delphi_sir_complainsalot/run.py:103]: line way too long

**Review by krivard — COMMENTED** [review]
Sir-CAL didn't lint before you picked it up, but I'd prefer not to make it worse :-P

File: sir_complainsalot/delphi_sir_complainsalot/run.py (hunk 1 above)
Comment (benjaminysmith) [reply sir_complainsalot/delphi_sir_complainsalot/run.py]: Eugh sorry -- I had pylint enabled in vscode but for some reason was not catching line length.

File: sir_complainsalot/delphi_sir_complainsalot/run.py (hunk 2 above)
Comment (benjaminysmith) [reply sir_complainsalot/delphi_sir_complainsalot/run.py]: Better??

**Review by krivard — APPROVED** [review]
xlnt

**Pull request merged**